giraf/giraf.go

```diff
@@ -14,6 +14,7 @@ type Giraf struct {
 	QName     string
 	QStart    int
 	QEnd      int
+	Flag      uint16
```

Comment:
Why not make these separate bool variables?  More space?  Even if we smash them together in the encoding for the file, maybe it makes sense to keep them separate in the data struct?  I did see that you have nice functions for accessing the individual bits, so maybe that is enough to keep it friendly?

Reply:
We could just make them bools in the struct and just encode them into a flag when we write to file i suppose. The only downside I see is a larger memory footprint (2 bytes for the flag vs 16 bytes for 16 bools). On the scale of the whole giraf struct an extra 14 bytes probably isn't much. I could go either way.